pkgs/historical-modules/default.nix

```diff
@@ -11,106 +11,193 @@ let
+        # .runners.runghc.displayVersion = "9.2.8";
+        displayVersion = "9";
+      };
     }
     {
       moduleId = "haskell-ghc9.4";
       commit = "4d6f7cd9fd685a3319ac7b6e3fc0789b430d6289";
-      displayVersion = "9.4.8";
+      overrides = {
+        # /nix/store/kkaxlri9s0bb7jydb1kpqzi2sjkf4rmj-replit-module-haskell-ghc9.4
+        # .runners.runghc.displayVersion = "9.4.8";
+        displayVersion = "9";
+      };
     }
     {
       moduleId = "elixir-1_15";
       commit = "4d6f7cd9fd685a3319ac7b6e3fc0789b430d6289";
-      displayVersion = "1.15.7";
+      overrides = {
+        # /nix/store/8nggvav01q7f1094m485nnd32094j522-replit-module-elixir-1_15
+        # .runners.elixir.displayVersion = "1.15.7";
+        displayVersion = "1";
+      };
     }
     {
       moduleId = "nodejs-14";
       commit = "f4cd419a646009297c049a2f1eec434381e08f13";
-      displayVersion = "14.21.1";
+      overrides = {
+        displayVersion = "14";
+        runners = {
+          nodeJS = {
+            displayVersion = "14.21.1";
+          };
+        };
+      };
     }
     {
       moduleId = "nodejs-16";
       commit = "f4cd419a646009297c049a2f1eec434381e08f13";
-      displayVersion = "16.18.1";
+      overrides = {
+        displayVersion = "16";
+        runners = {
+          nodeJS = {
+            displayVersion = "16.18.1";
+          };
+        };
+      };
     }
     {
       moduleId = "nodejs-19";
       commit = "f4cd419a646009297c049a2f1eec434381e08f13";
-      displayVersion = "19.1.0";
+      overrides = {
+        displayVersion = "19";
+        runners = {
+          nodeJS = {
+            displayVersion = "19.1.0";
+          };
+        };
+      };
     }
     {
       moduleId = "php-8.1";
       commit = "0b7a60667d2c29f2686211e952924a9693916a20";
-      displayVersion = "8.1.20";
+      overrides = {
+        # /nix/store/xc89rlwnmg5vh66hvrv1saq6md6kp1g5-replit-module-php-8.1
+        # .runners.php.displayVersion = "8.1.20";
+        displayVersion = "8";
+      };
     }
     {
       moduleId = "r-4.2";
       commit = "1e1bb663068482cdb7c04bf585daed00205c0140";
-      displayVersion = "4.2.3";
+      overrides = {
+        # /nix/store/knizlzgmqgm488llvxxaai6kjb17mhm2-replit-module-r-4.2
+        # .runners.r.displayVersion = "4.2.3";
+        displayVersion = "4";
+      };
     }
     {
       moduleId = "swift-5.6";
       commit = "c48c43c6c698223ed3ce2abc5a2d708735a77d5b";
-      displayVersion = "5.6.2";
+      overrides = {
+        # /nix/store/f96l8blsvpx1dbgnxw2x611hhzbm199l-replit-module-swift-5.6
+        # .runners.swift.displayVersion = "5.6.2";
+        displayVersion = "5";
+      };
     }
     {
       moduleId = "vue-node-18";
       commit = "3ea4bcdbdc3c5e3c09b37b07edcd61781f9695f7";
-      displayVersion = "18.18.2";
+      overrides = {
+        # /nix/store/623sn0cy54r5csab3d9fmg55di6r4dd3-replit-module-vue-node-18
+        # .runners["dev-runner"].displayVersion = "18.18.2";
```

Comment:
Why are some of these commented out?

Reply:
I spot-checked and it didn't seem as though they were used, I figured I'd just fix the ones that I actually need right now and we can add the rest of the specific versions later if it turns out they're useful. I did the legwork of correlating the values and keys with the comments, so it should be easier in the next pass.

Reply:
ok